packages/react-inbox/src/components/Messages2.0/Header.tsx

```diff
@@ -322,12 +322,12 @@ const Header: React.FunctionComponent<IHeaderProps> = ({
         </HeadingContainer>
       )}
       <div className="actions">
-        {messages.length > 0 && unreadMessageCount && (
+        {messages.length > 0 && unreadMessageCount ? (
           <MarkAllRead
             label={labels?.markAllAsRead}
             onClick={() => markAllAsRead()}
           />
-        )}
+        ) : null}
```

Comment:
what is the need for this?

Reply:
it was rendering a 0 vs not rendering anything

Reply:
that's v odd. `unreadMessageCount` being 0 should evaluate to falsy. Does the `MarkAllRead` component have some other value in it?

Reply:
jk https://stackoverflow.com/questions/53048037/react-showing-0-instead-of-nothing-with-short-circuit-conditional-component